ssl/quic/quic_sstream.c

```diff
@@ -349,7 +349,7 @@ static void qss_cull(QUIC_SSTREAM *qss)
      * can only cull contiguous areas at the start of the ring buffer anyway.
      */
     if (h != NULL)
-        ring_buf_cpop_range(&qss->ring_buf, h->range.start, h->range.end);
+        ring_buf_cpop_range(&qss->ring_buf, h->range.start, h->range.end, 0);
```

Comment:
Shouldn't we be cleansing send streams also?

Reply:
We don't cleanse them on TLS connections when SSL_OP_CLEANSE_PLAINTEXT is set either.

Reply:
Ah, there is in-place encrypt in case of TLS. I did not know that. In that case yeah, we should cleanse the send stream buffers too once acked.